source/solvers/vof_assemblers.cc

```diff
@@ -166,7 +168,9 @@ VOFAssemblerCore<dim>::assemble_rhs(VOFScratchData<dim> &      scratch_data,
   const auto method = this->simulation_control->get_assembly_method();
 
   // Add a small diffusivity, used in the context of the wetting mechanism
-  const double diffusivity = this->vof_parameters.peeling_wetting.diffusivity;
+  double diffusivity = 0.;
+  if (this->vof_parameters.peeling_wetting.enable)
+    diffusivity = this->vof_parameters.peeling_wetting.diffusivity;
```

Comment:
Are you sure you want to enable diffusivity only for peeling and wetting? it could be useful for other stuff. Then you could put the default value to 0 and it would have no effect